src/System.Diagnostics.Process/tests/ProcessStreamReadTests.cs

```diff
@@ -88,6 +89,53 @@ public void TestAsyncOutputStream()
             }
         }
 
+        [Fact]
+        // On this test RemotelyInvokable.WriteLoop never returns, we cannot use RemotelyInvokable.WriteLoopUapCmd workaround
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void TestAsyncOutputStream_BeginCancelBegin_OutputReadLine()
```

Comment:
This should be Outerloop, given the long sleeps involved.

Reply:
test now run for no more than 3.x seconds...do we run in outerloop  again?

Reply:
Yes, tests that run for seconds should be Outerloop.

Reply:
done